### ISSUE TITLE
editing issues from context menu on issues page throws error 500

### ISSUE THREAD

**Issue by leoniscsem** [MayamaTakeshi/redmine_rt#8]
I have upgraded to the latest release of the plugin, but I am still on `Redmine 4.2.1`, so I don't know if this is related to the core. Reproduce:

- go to issues overview page
- right click on issue
- change status from "new" to "in progress"
- error 500

production.log
```shell
Started POST "/issues/bulk_update?back_url=%2Fprojects%2Fdemo%2Fissues%3Fc%255B%255D%3Dtracker%26c%255B%255D%3Dstatus%26c%255B%255D%3Dpriority%26c%255B%255D%3Dsubject%26c%255B%255D%3Dassigned_to%26c%255B%255D%3Dupdated_on%26f%255B%255D%3Dstatus_id%26f%255B%255D%3Dsubproject_id%26f%255B%255D%3D%26group_by%3D%26op%255Bstatus_id%255D%3Do%26op%255Bsubproject_id%255D%3D%2521%26set_filter%3D1%26sort%3Did%253Adesc%26t%255B%255D%3D%26utf8%3D%25E2%259C%2593%26v%255Bsubproject_id%255D%255B%255D%3D7&ids%5B%5D=28&issue%5Bdone_ratio%5D=10" for <IP-ADDRESS> at 2022-01-04 11:37:56 +0800
Processing by IssuesController#bulk_update as HTML
  Parameters: {"authenticity_token"=>"ks2bBB1SSQ3TyilKAT1zHO-dPbOwfuBAMFOCaOq1zIombJ82iaZHlmFiwLXv6wmA3wa98WJ0gDPoJ5f9FsROvw", "back_url"=>"/projects/demo/issues?c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=assigned_to&c%5B%5D=updated_on&f%5B%5D=status_id&f%5B%5D=subproject_id&f%5B%5D=&group_by=&op%5Bstatus_id%5D=o&op%5Bsubproject_id%5D=%21&set_filter=1&sort=id%3Adesc&t%5B%5D=&utf8=%E2%9C%93&v%5Bsubproject_id%5D%5B%5D=7", "ids"=>["28"], "issue"=>{"done_ratio"=>"10"}}
  Current user: christopher (id=5)
Completed 500 Internal Server Error in 143ms (ActiveRecord: 26.0ms)
  
NoMethodError (undefined method `[]' for nil:NilClass):
  
plugins/redmine_rt/lib/redmine_rt/journal_patch.rb:29:in `handle_journal_after_create'
app/models/journal.rb:100:in `save'
app/models/issue.rb:1982:in `create_journal'
app/models/issue.rb:222:in `create_or_update'
app/controllers/issues_controller.rb:381:in `block in bulk_update'
app/controllers/issues_controller.rb:365:in `each'
app/controllers/issues_controller.rb:365:in `bulk_update'
lib/redmine/sudo_mode.rb:61:in `sudo_mode'
```

Environment
```
Environment:
  Redmine version                4.2.1.stable
  Ruby version                   2.7.5-p203 (2021-11-24) [x86_64-linux-gnu]
  Rails version                  5.2.5
  Environment                    production
  Database adapter               PostgreSQL
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
SCM:
  Git                            2.17.1
  Filesystem                     
Redmine plugins:
  dashboard                      1.0.11
  redmine_agile                  1.6.1
  redmine_base_deface            1.5.3
  redmine_callout_macro          1.0.0
  redmine_checklists             3.1.20
  redmine_ics_export             4.0.1.dev
  redmine_issue_dynamic_edit     0.7.2
  redmine_mentions               0.0.1
  redmine_messenger              1.0.9
  redmine_more_code              1.0.1
  redmine_rt                     1.0.0
```

**Comment by MayamaTakeshi** [comment]
OK. Confirmed here (it happens with latest redmine 4.2.3 too).
I will try to work on it tomorrow.

**Comment by MayamaTakeshi** [comment]
It turned out it was a simple problem: addition of journal without notes (comments) caused by that kind of user action was not being handled properly. It was solved (tested with both redmine 4.1.5 and 4.2.3).
Please pull latest changes.

**Comment by leoniscsem** [comment]
Fantastic, I just pulled and it works flawlessly.
